src/main/java/io/milvus/client/AbstractMilvusGrpcClient.java

```diff
@@ -37,7 +38,325 @@
                 .orElse(false);
 
         return R.success(aBoolean);
+    }
+
+    @Override
+    public R<FlushResponse> flush(String collectionName, String dbName) {
+        return flush(Collections.singletonList(collectionName), dbName);
+    }
+
+    @Override
+    public R<FlushResponse> flush(String collectionName) {
+        return flush(Collections.singletonList(collectionName), "");
+
+    }
+
+    @Override
+    public R<FlushResponse> flush(List<String> collectionNames) {
+        return flush(collectionNames, "");
+
+    }
+
+    @Override
+    public R<FlushResponse> flush(List<String> collectionNames, String dbName) {
+        MsgBase msgBase = MsgBase.newBuilder().setMsgType(MsgType.Flush).build();
+        FlushRequest.Builder builder = FlushRequest.newBuilder().setBase(msgBase).setDbName(dbName);
+        collectionNames.forEach(builder::addCollectionNames);
+        FlushRequest flushRequest = builder.build();
+        FlushResponse flush = null;
+        try {
+            flush = blockingStub().flush(flushRequest);
+        } catch (Exception e) {
+            return R.failed(e);
+        }
+        return R.success(flush);
+    }
+
+    @Override
+    public R<MutationResult> delete(DeleteParam deleteParam) {
+
+        DeleteRequest deleteRequest = DeleteRequest.newBuilder()
+                .setBase(MsgBase.newBuilder().setMsgType(MsgType.Delete).build())
+                .setCollectionName(deleteParam.getCollectionName())
+                .setPartitionName(deleteParam.getPartitionName())
+                .build();
+
+        MutationResult delete = null;
+        try {
+            delete = blockingStub().delete(deleteRequest);
+        } catch (Exception e) {
+            return R.failed(e);
+        }
+        return R.success(delete);
+    }
+
+    @Override
+    public R<MutationResult> insert(InsertParam insertParam) {
+        //check params : two steps
+        // 1、check sdk incoming params
+        String collectionName = insertParam.getCollectionName();
+        String partitionName = insertParam.getPartitionName();
+        int fieldNum = insertParam.getFieldNum();
+        List<String> fieldNames = insertParam.getFieldNames();
+        List<DataType> dataTypes = insertParam.getDataTypes();
+        List<List<?>> fieldValues = insertParam.getFieldValues();
+        Integer filedNameSize = Optional.ofNullable(fieldNames).map(List::size).orElse(0);
+        Integer dTypeSize = Optional.ofNullable(dataTypes).map(List::size).orElse(0);
+        int fieldValueSize = Optional.ofNullable(fieldValues).map(List::size).orElse(0);
+        if (fieldNum != fieldValueSize || fieldNum != filedNameSize || fieldNum != dTypeSize) {
+            throw new ParamException("size is illegal");
```

Comment:
print more detailed information about what field is illegal, that helps user to do self debug

Reply:
this need DDL. this exception will add int next step. todo